ultraplot/axes/geo.py

```diff
@@ -1290,8 +1686,8 @@ def _update_major_gridlines(
         )
         gl.xformatter = self._lonaxis.get_major_formatter()
         gl.yformatter = self._lataxis.get_major_formatter()
-        self.xaxis.set_major_formatter(mticker.NullFormatter())
-        self.yaxis.set_major_formatter(mticker.NullFormatter())
+        # self.xaxis.set_major_formatter(mticker.NullFormatter())
+        # self.yaxis.set_major_formatter(mticker.NullFormatter())
```

Comment:
If this code is not needed, it should be removed. Otherwise, we need to add it back.